pkg/services/sqlstore/user.go

```diff
@@ -43,7 +43,7 @@ func getOrgIdForNewUser(cmd *m.CreateUserCommand, sess *DBSession) (int64, error
 
 	if setting.AutoAssignOrg {
 		// right now auto assign to org with id 1
-		has, err := sess.Where("id=?", 1).Get(&org)
+		has, err := sess.Where("id=?", setting.AutoAssignOrgId).Get(&org)
```

Comment:
Needs to handle the case when the specified org doesn't exist, right now it'll always create org id 1.  I suspect that the better way to handle this would be to make sure the main org exists at startup instead and remove the automatic creation here.

Reply:
Are you sure this always creates the org? I assume has will be non-null and it'll just return the org id of the already existing organization.

Reply:
If the org isn't found then `has` will be false and it'll try to create an org lower down in this function

Reply:
I see what you're going for, what about setting 
```org.Id = setting.AutoAssignOrgId```
In this scope?